### DISCUSSION TITLE
Right click on products is not working

### DISCUSSION THREAD

**Discussion by RajeshNakkanaboina** [clientIO/joint#2259] [Q&A]
### Introduction

Hi Team,

Right-clicking on items when drawing the graph does not work. Would someone kindly assist me? 

Thanks in advance

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
If you meant that the native context menu is not showing, then use:
```js
paper.options.preventContextMenu = false;
```
It is disabled by default.
